docs/examples/python/direct_collocation.py

```diff
@@ -133,7 +133,7 @@
        ubg.append([0, 0])
 
        # Add contribution to the end state
-       Xk_end = Xk_end + D[j]*Xc[j-1];
+       Xk_end = Xk_end + D[j]*Xc[j-1]
```

Comment:
shouldnt this be instead
```suggestion
       Xk_end = Xk_end + D[k]*Xc[j-1]
```

Reply:
Hi @ottarg , thanks for the review.

The suggested fix does not appear to be correct, though.